### DISCUSSION TITLE
Issues with BlueSCSI v2 on Amiga 3000 external DB-25 port

### DISCUSSION THREAD

**Discussion by jdieguez** [BlueSCSI/BlueSCSI-v2#141] [Help] [answered]
I have an internal blueSCSI v2  (50 pin ribbon cable version) and an external BlueSCSIv2 (DB-25).
Both work trouble free with a Mac LCIII (connected to the internal and external SCSI ports respectively).

When I try to connect them to an Amiga 3000, the internal version works fine if connected to the internal SCSI, but I'm having some issues when trying to use the external DB-25 port. Note that I have a terminated HDD inside the internal amiga SCSI (providing termination on that end) and the termination resistor packs are also removed from the amiga PCB (to allow connection of external devices).

- I've tried to attach my BlusSCSIs DB-25 to the DB25 port. It won't work at all.
- I've tried to attach my BlueSCSI internal to the DB-25 port, using an external enclosure (which converts DB-25 to centronics, and then to a standard ribbon cable, which I connect to the blueSCSI). The strange thing here is that if I try to enable the termination jumper on the blueSCSI, it does not work. However, if I disable the jumper and provide a passive SCSI terminator at the end of the chain, it works (I did not try with an active terminator because I don't have one).

I guess the root cause for the onboard termination not working on either device should be the same. It if would be fixed on the internal version, then that would also help getting the external version to work.

**Comment by svenson-a** [comment]
Hi Francisco,
i am running the BlueSCSI V2 (so called Desktop version) on an Amiga  A3000D. I attached it externally, because of the fact i am running a SCSI2SD internally. Have a look at the pictures for the external setup. "Termination On" is jumpered on the BlueSCSI, as well as "Power On".
It works flawlessly so far on my A3000D. Internal SCSI terminators are removed from the A3000 mainboard.
Which SCSI controller chip do you have on the A3000 mainboard? I use an AMD AM33C93A.

This is my bluescsi.ini:
```
[SCSI]
WiFiSSID=my_ssid
WiFiPassword=my_password
System="Generic"
EnableSCSI2=0 ; Off
PrefetchBytes=0
Debug=0
[SCSI2]
Vendor=Blue
Product=Bluedrive
Version=1.0

``` 
Kind regards, Svenson

![IMG_8919](https://github.com/BlueSCSI/BlueSCSI-v2/assets/160474039/9f995a99-f67a-42a6-bc19-53d5a677503d)
![IMG_8918](https://github.com/BlueSCSI/BlueSCSI-v2/assets/160474039/a1d4f51e-cad1-4e58-98f5-d8e3f2f77a85)

**Reply by jdieguez** [reply]
thanks! This is valuable information.
The main difference with my setup is that my internal hard drive is an old school 4GB Quantum Fireball with onboard termination.
Perhaps the termination scheme in this drive is different and needs a passive termination on the other end in order for the external drives to work.

As I have both internal (desktop) and an external (DB25) BlusSCSI drives, I will try to replace the Quantum HDD with the Deskptop BlueSCSI, and check if the DB25 BlueSCSI now works. 

Another thing I will try is to remote the internal drive altogether and put there an active terminator instead (which I have just purchased).

Will report afterwards.

Thanks again

**Comment by androda** [comment, marked as answer]
> I guess the root cause for the onboard termination not working on either device should be the same. It if would be fixed on the internal version, then that would also help getting the external version to work.

From my perspective it could be a power issue.  The termination power line is mostly intended to power SCSI terminators (as the name implies).  But BlueSCSI can also run from the termpower line, which adds some extra current draw for the Pico and SD card and such.

Where possible, when running two BlueSCSI or other SCSI drive emulator units it is good to apply power to the floppy connector on at least one of them to decrease load on the termpower line.  The USB power connector can also be used for external power.

**Comment by jdieguez** [comment]
Thanks for the replies guys. I did some more tests and it is not a termination issue but a power issue. I plugged a powerbank to the microUSB in the pico of the DB25 and now it works...

What is strange is that it cannot be powered from the termpower line alone. The internal drive on my A3000 (either the quantum HDD or the blusescsi desktop) uses power from the molex connector, so there should not be much draw from termpower line. I also measured pin25 (termpower) of the DB25 and it is 4.85V. And to my understanding, the only thing between this pin and the PSU is a diode.

This same BlueSCSI DB25 works perfectly in my mac LCIII without the need for external power.

**Comment by jdieguez** [comment]
I have made some progress.

I have measured the VSYS voltage on the Raspberry pico of the BlueSCSI DB-25 (with the A3000 turned on).
With power coming via microUSB (power bank) I get close to 5V and the blueSCSI works. With nothing connected to the microUSB, it fluctuates between 0.7V and 1.3V, not enough to power the device.

I have done some more testing and I have seen that diode D800 on the motherboard is shorted. This diode connects the +5V from the A3000 PSU to pin 25 (TPWR) on the DB25 connector.
I will replace the diode and hopefully the termpower on the DB25 port will work properly afterwards

![IMG_1581](https://github.com/BlueSCSI/BlueSCSI-v2/assets/20295989/dfaaf5ad-fca0-4ada-850a-c3424de7fbf8)
![IMG_1583](https://github.com/BlueSCSI/BlueSCSI-v2/assets/20295989/ab25156a-db58-49c0-85af-5beb96235afd)

**Comment by jdieguez** [comment]
Took the Amiga apart to work on the motherboard.

Not only the diode was shorted, but there was a bit of a mess in the trace connecting the cathode to the DB25 pin 25 through a via). Something nasty happened in the past... 

So, I replaced the diode with a new one (Schottky), and connected the cathode to pin 25.

Voila! The BlueSCSI now works when powered from termpower! :)
![IMG_1589](https://github.com/BlueSCSI/BlueSCSI-v2/assets/20295989/91de3eb4-9bad-441e-9870-8273c036c5ed)
![IMG_1593](https://github.com/BlueSCSI/BlueSCSI-v2/assets/20295989/5de3aac5-cb7a-4878-9097-72c74e249231)
![IMG_1595](https://github.com/BlueSCSI/BlueSCSI-v2/assets/20295989/89559dec-5af9-4bea-b491-dd0c3985d84c)

**Comment by svenson-a** [comment]
Hi Francisco,
this is a great success! Congratulations for solving it!
Kind regards, Svenson

**Reply by salocinx** [reply]
Be aware that diode D800 was not only populated the wrong direction on some A3000D mainboards, but they also printed the silkscreen wrongly! I recommend to double check in order to avoid further problems. Regards, Nicolas.

**Reply by jdieguez** [reply]
thanks, I was already aware of this :)

I double checked that the anode is connected to the PSU +5V line, and the cathode to pin 25 in the DB25 connector. Power is going out now and is able to power the BlueSCSI (the BlueSCSI would not power up otherwise).

cheers
